libraries/AP_BattMonitor/AP_BattMonitor_UAVCAN.cpp

```diff
@@ -43,6 +45,15 @@ void AP_BattMonitor_UAVCAN::subscribe_msgs(AP_UAVCAN* ap_uavcan)
         AP_HAL::panic("UAVCAN BatteryInfo subscriber start problem\n\r");
         return;
     }
+
+    uavcan::Subscriber<ardupilot::equipment::power::BatteryInfoAux, BattInfoAuxCb> *battinfo_aux_listener;
+    battinfo_aux_listener = new uavcan::Subscriber<ardupilot::equipment::power::BatteryInfoAux, BattInfoAuxCb>(*node);
+    // Backend Msg Handler
+    const int battinfo_aux_listener_res = battinfo_aux_listener->start(BattInfoAuxCb(ap_uavcan, &handle_battery_info_aux_trampoline));
+    if (battinfo_aux_listener_res < 0) {
+        AP_HAL::panic("UAVCAN BatteryInfoAux subscriber start problem\n\r");
```

Comment:
This will cross over with one of Sid's PRs which changes this to a different call.

Reply:
Which PR is it specifically?

Reply:
https://github.com/ArduPilot/ardupilot/pull/18694